### ISSUE TITLE
unknown URI: faust://document/bohnenkamp/H_P182

### ISSUE THREAD

**Issue by sandrakrause** [faustedition/faust-gen-html#569]
Laut Bohnenkamp ist P182 die Fortsetzung von P181.
Wie soll damit umgegangen werden?

**Comment by dietmarpravida** [comment]
Kann man eigentlich issues von gen-html nach xml ziehen?

**Comment by dietmarpravida** [comment]
P181 und 182 gehören zusammen = IV H.20; bohnenkamp/H_P182  gibt es nicht: ein eigener Datierungseintrag zu P182 kann gelöscht werden, wenn es zu IV H.20 / bohnenkamp/H_P181 einen gibt

**Comment by dietmarpravida** [comment]
@gerritbruening Ein anderes Probem ist, ob und wie die Zusammengehörigkeit von H P181 und der Vorderseite von IV H.20 (mit P182) ausgedrückt wird.

**Comment by gerritbruening** [comment]
> Kann man eigentlich issues von gen-html nach xml ziehen?

Nein, das geht glaube ich nicht so richtig. Zur Not steht aber in faust-gen-html das label `c:xml-source` zur Verfügung; die entsprechenden Issues bitte damit kennzeichnen.

**Comment by gerritbruening** [comment]
> Ein anderes Probem ist, ob und wie die Zusammengehörigkeit von H P181 und der Vorderseite von IV H.20 (mit P182) ausgedrückt wird.

Überlieferungsträgerübergreifende (s. edit. Grundbegr. :-) Inskriptionen, habt ihr das seinerzeit modelliert?

**Comment by gerritbruening** [comment]
Achso. Aber wie auch immer, es hängt wohl davon ab, welche Einheiten datiert werden können und wie diese definiert sind. Ob es also z.b. Inspektionen über Dokumente hinweg geben kann.
